src/tabs/tabset.component.html

```diff
@@ -1,4 +1,4 @@
-<ul class="nav" [ngClass]="classMap" (click)="$event.preventDefault()">
+<ul *ngIf="isKeysAllowed" class="nav" [ngClass]="classMap" (click)="$event.preventDefault()">
```

Comment:
I believe disabling `keyNavActions` handler would be a better option than template duplication
performance wise too

Reply:
!

Reply:
!